docs/tutorials/getting-started.md

```diff
@@ -42,7 +42,47 @@ with the tests.
 
 ## Run the test suite
 
-🚧 **TODO**
+#### Redash Setup
+The Redash instance must be setup before running the tests. Start the images ```docker-compose up -d```. This will download the images if you don't have them.
+
+Run these following commands to setup Redash:
+1. ```docker-compose run --rm server create_db```
+2. ```docker-compose run --rm postgres psql -h postgres -U postgres -c "create database tests"```
+3. ```docker-compose run --rm server /app/manage.py users create_root root@example.com "rootuser" --password "IAMROOT" --org default```
+4. ```docker-compose run --rm server /app/manage.py ds new "ui-tests" --type "url" --options '{"title": "uitests"}'```
+
+You can visit ```127.0.0.1:5000```, or ```localhost:5000``` and you should see a Redash login page.
+___
+The tests can be run within a custom built docker image, or on your local machine using Firefox.
+
+#### Docker Test
+Build the redash-ui-tests with this command:
+```
+docker build -t "redash-ui-tests:latest"
+```
+The run the tests:
+```
+docker run --net="host" --env REDASH_SERVER_URL=http://127.0.0.1:5000 redash-ui-tests:latest
+```
+If there are failures you can view the html report by first running this command
+```
+docker cp ui-tests:/home/user/src/report.html ./report.html
+```
+and open the ```report.html``` in the web browser.
+
+#### Local test
```

Comment:
I feel like we can remove this section. In my experience it's better to show newcomers only one way how to do things and make sure that it works reliably. Using Docker is the preferred way, I'd say.

Reply:
But what about someone more experienced? I think having both works and if someone doesn't understand one way, they'd use the other set of instructions.

Reply:
I am OK with describing alternatives in a different section of the documentation but not in this tutorial. We could link to [how-to guides](https://redash-ui-tests.readthedocs.io/en/latest/) from here.